### PR TITLE
fix(core): use correct SSH port

### PR DESCRIPTION
**Pull request by CPbN**
Hi,

## Description

Using a non default SSH port on a Remote Server leads to the following in `centcore.log`,
on the Central Server :

```
2019-06-18 16:54:19 - INFO - Export files on poller "remote-server" (32)
2019-06-18 16:54:28 - INFO - Result : [sender] io timeout after 6 seconds -- exiting
rsync error: timeout in data send/receive (code 30) at io.c(195) [sender=3.1.2]
```

This is due to a non valid usage of the port option of the rsync command.
While here, I found other SSH and rsync commands not using the configured port, so I fixed them.

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 19.04.x
- [x] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

Use a non default SSH port for your Remote Server and verify it is correctly used by centcore.

Thank you 👍

Edit : issue still present in 19.04.2.

### CODE REVIEW

File: lib/perl/centreon/script/centcore.pm [hunk 1]
```diff
@@ -775,7 +775,7 @@ sub sendExportFile($){
     # Send data with rSync
     $self->{logger}->writeLogInfo('Export files on poller "' . $server_info->{name} . '" (' . $id . ')');
 
-    $cmd = "$self->{rsync} -ra --port=$port $origin $dest 2>&1";
+    $cmd = "$self->{rsync} -ra -e '$self->{ssh} -o port=$port' $origin $dest 2>&1";
```
Comment (lpinsivy) [inline lib/perl/centreon/script/centcore.pm]: The man of rsync indicates:           --port=PORT             specify double-colon alternate port number
I don't understand your change

Comment (CPbN) [reply lib/perl/centreon/script/centcore.pm]: Yes it's a little bit tricky.
The `--port` option of rsync is used to specify the rsync listening port, when rsync is used in daemon mode (without SSH tunnel).
But here, as we use SSH as the transport tunnel, we have to indicate the destination port to SSH itself.
Thus, we have to use `-e` rsync option to tune the SSH call, simply giving it `-o port=...`.

**Comment by lpinsivy** [comment]
I made a test and it works great ;)

**Comment by CPbN** [comment]
Perfect, many thanks for your approval :+1:

**Comment by adr-mo** [comment]
**The conflicts must be resolved. Otherwise I can't start validation**

Thanks

**Comment by CPbN** [comment]
@adr-mo I did it in the 3rd commit 2 days ago ? :)

**Comment by adr-mo** [comment]
Okay, because I've tried to rebase it just a few minutes ago and found that it could not be done directly (conflicts).

I am saying that because I know that @lpinsivy made corrections on the file that you modified.

Could you please double check ? :+1: 

Regards

**Comment by CPbN** [comment]
Github says itself here there's no conflict with base / master branch ?
You're right @lpinsivy made modifications on one of these files, this is why I then pushed the 3rd commit of this PR (https://github.com/centreon/centreon/pull/7630/commits/dc1a7209e4f3b4622aac8d3882dc4751d403a40a).
I'm not sure how to help here then :-/

**Comment by lpinsivy** [comment]
New PR made to solved conflicts

**Pull request closed without merging**